aws/autoscaling.go

```diff
@@ -20,33 +20,24 @@ import (
 
 type AutoScalingGroup struct {
 	AWSResource
+	autoscaling.Group
 
 	// autoscaling.Instance exposes minimal info
 	Instances []reapable.ID
-
-	AutoScalingGroupARN     string
-	CreatedTime             time.Time
-	MaxSize                 int64
-	MinSize                 int64
-	DesiredCapacity         int64
-	LaunchConfigurationName string
 }
 
 func NewAutoScalingGroup(region string, asg *autoscaling.Group) *AutoScalingGroup {
+	if asg == nil {
```

Comment:
This should have been in in the first place - make sure the pointer to the AWS type is not nil